includes/Renderer/Stories/Generic_Renderer.php

```diff
@@ -81,10 +81,13 @@ public function render( array $args = [] ): string {
 		<div class="<?php echo esc_attr( $container_classes ); ?>" data-id="<?php echo esc_attr( (string) $this->instance_id ); ?>">
 			<div class="web-stories-list__inner-wrapper" style="<?php echo esc_attr( $container_styles ); ?>">
 				<?php
-				foreach ( $this->stories as $story ) {
-					$this->render_single_story_content();
-					$this->next();
-				}
+				array_map(
```

Comment:
Ditto here about `array_map` vs `foreach`

Reply:
Explain in other place. Made these changes manually.